client/packages/plugins/backendCommon/utils.ts

```diff
@@ -21,6 +21,7 @@ export const sqlQuery = <K extends string>(
   sqlStatement: string
 ): Record<K, any>[] => {
   const adjustedStatement = wrapSql(fields, sqlStatement);
+  // log(adjustedStatement);
```

Comment:
can remove :)

Reply:
I like this one, sorry, can i keep ? i had to log these to debug a few times